core/api/views/chemicals.py

```diff
@@ -1,17 +1,19 @@
 from rest_framework import mixins, generics
-from core.api.serializers import GroupSubstanceSerializer
 
+from core.api.serializers import GroupSubstanceSerializer
+from core.api.serializers.chemicals import BlendSerializer
+from core.models.blend import Blend
 from core.models.group import Group
 
 
 class GroupSubstancesListAPIView(mixins.ListModelMixin, generics.GenericAPIView):
```

Comment:
I wonder if we really need substances to be grouped by annex/group in this endpoint. Wouldn't a flat list of substances, similar to the blends endpoint, be easier to work with?

Reply:
@iuliansafta

Reply:
I requested to be grouped by annex since it's easier to work with on the front end. I have some places where substances are displayed in a select box and visually is better to be grouped.